src/status.rs

```diff
@@ -40,6 +40,8 @@ impl Spacefed {
 #[derive(Serialize, Deserialize, Default, Debug, Clone, PartialEq, Eq)]
 pub struct Icon {
     pub open: String,
+    // FIXME: rename this field with the next major version
+    #[serde(rename = "closed")]
```

Comment:
The next release will be with breaking changes anyways, so we can just rename the field.

Reply:
Alright!